### PR TITLE
Change Jira product name case

### PR DESCRIPTION
**Pull request by adamdmharvey**
Per [this announcement in Sept 2017](https://community.atlassian.com/t5/Feedback-Forum-articles/A-new-look-for-Atlassian/ba-p/638077), Atlassian officially rebranded JIRA to Jira (proper case).

> "Note that JIRA (all caps) has also become Jira (initial cap) as part of this update. "

This just changes some of the general references to the plugin to align with this, without (I believe) affecting any key functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,54 +38,54 @@ Also make sure that CAPTCHA is not triggered for your user as this will
...
-write access to the JIRA (say you have a Jenkins build for one of the
+write access to the Jira (say you have a Jenkins build for one of the
 Apache commons project that you depend on), so that's why this is
 optional.  
 
-And the following screen shows how JIRA issue is updated:
+And the following screen shows how Jira issue is updated:
 
-![](docs/images/JIRA_Comments.jpg)
+![](docs/images/Jira_Comments.jpg)
```
Comment (warden) [inline README.md]: Filename needs to be changed as well

File: README.md [hunk 2]
```diff
@@ -112,38 +112,38 @@ Sample usage of generated Release Notes:
...
+| **-Dhudson.plugins.jira.JiraMailAddressResolver.disabled=true** | Use to disable resolving user email from Jira usernames. Currently there is no option for this in UI. |
 
 #### Related Resources
 
 -   Check also the Marvelution [Jira Hudson Integration](http://www.marvelution.com/atlassian/jira-hudson-integration/)
-    which provides a two-way solution Hudson-\>JIRA and JIRA-\>Hudson
--   [Hudson integration for JIRA](https://plugins.atlassian.com/plugin/details/11858) adds Hudson information to JIRA.
--   The [Subversion JIRA plugin](https://studio.plugins.atlassian.com/wiki/display/SVN/Subversion+JIRA+plugin) also allows recording of scm changes to JIRA issues (for other SCMs
+    which provides a two-way solution Hudson-\>Jira and Jira-\>Hudson
+-   [Hudson integration for Jira](https://plugins.atlassian.com/plugin/details/11858) adds Hudson information to Jira.
```
Comment (warden) [inline README.md]: This is archived and can be removed

File: README.md [hunk 3]
```diff
@@ -112,38 +112,38 @@ Sample usage of generated Release Notes:
...
+-   The [Subversion Jira plugin](https://studio.plugins.atlassian.com/wiki/display/SVN/Subversion+JIRA+plugin) also allows recording of scm changes to Jira issues (for other SCMs
```
Comment (warden) [inline README.md]: This link doesn't work anymore so I guess this also can be removed

File: README.md (hunk 1 above)
Comment (adamdmharvey) [reply README.md]: done!

**Comment by adamdmharvey** [comment]
@warden Indeed, a very good suggestion! I had intended to just work through the repo doco, but will expand this to entail the actual display of the plugin itself.  Give me a bit to work through and cleanly do this.  Thanks!!

**Comment by adamdmharvey** [comment]
Note one place I purposely did not change it was in the `pom.xml`, figuring this entry has more to do with tying into the Jenkins issue tracking mechanism?

https://github.com/jenkinsci/jira-plugin/blob/db54218006458a02d4bdc5276321ce3299f2057a/pom.xml#L75-L78

**Comment by adamdmharvey** [comment]
Believe this should be ready for review. 👍 Open to any and all feedback and good suggestion re: the msgs.

File: src/main/java/hudson/plugins/jira/selector/perforce/P4JobIssueSelector.java [hunk 4]
```diff
@@ -44,10 +44,10 @@ public P4JobIssueSelector() {
 
     @Override
     protected void addJobIdsFromChangeLog(Run<?, ?> build, JiraSite site, TaskListener listener, Set<String> issueIds) {
-        getLogger().finer("Searching for JIRA issues in perforce jobs in " + build);
+        getLogger().finer("Searching for Jira issues in perforce jobs in " + build);
         for (ChangeLogSet<? extends Entry> set : RunScmChangeExtractor.getChanges(build)) {
             for (Entry change : set) {
-                getLogger().fine("Looking for JIRA IDs as Perforce Jobs in " + change.getMsg());
+                getLogger().fine("Looking for Jira as Perforce Jobs in " + change.getMsg());
```
Comment (warden) [inline src/main/java/hudson/plugins/jira/selector/perforce/P4JobIssueSelector.java]: Jira IDs

**Comment by warden** [comment]
@adamdmharvey thanks for the big effort!

**Pull request merged**